Header header
int32[3] left_wheel_counts   # Front to back
int32[3] right_wheel_counts  # Front to back
int32 base_rotation_counts
int32 wrist_rotation_counts
int32 gripper_counts

